runtime/browser/xwalk_content_browser_client.cc

```diff
@@ -275,6 +276,20 @@ void XWalkContentBrowserClient::AllowCertificateError(
                                   &cancel_request);
   if (cancel_request)
     *result = content::CERTIFICATE_REQUEST_RESULT_TYPE_DENY;
+#else
+  content::RenderFrameHost* render_frame_host =
+      content::RenderFrameHost::FromID(render_process_id, render_frame_id);
```

Comment:
In the next statement, 'WebContents::FromRenderFrameHost' is pretty strong, even feed  'nullptr' to him.